codalab/model/bundle_model.py

```diff
@@ -2234,19 +2234,30 @@ def get_user_time_quota_left(self, user_id):
     def get_user_parallel_run_quota_left(self, user_id):
         user_info = self.get_user_info(user_id)
         parallel_run_quota = user_info['parallel_run_quota']
+
+        # The following sqlalchemy query is equivalent to
+        # SELECT worker_run.run_uuid
+        # From worker_run JOIN worker
+        # ON worker.user_id=worker_run.user_id
+        # WHERE worker.tag IS NULL and worker.user_id = <user_id> and worker_run.user_id = <user_id>;
+        # Note: <user_id> is an input parameter
+        join = cl_worker_run.join(cl_worker, cl_worker_run.c.user_id == cl_worker.c.user_id)
```

Comment:
Can you rename this variable to a more descriptive name?

Reply:
Oh, I was actually following another outer join clause in our codebase at [bundle_model.py](https://github.com/codalab/codalab-worksheets/blob/master/codalab/model/bundle_model.py#L619). Please let me know if we want to change both of them to something else. Maybe we could rename them to be something containing tables name?

Reply:
I think in these cases it's better to try make the new code better styled then stay consistent. Feel free to update the other names as well, but looking at your link that looks like an annoying place to edit.